static/bidder-info/onemobile.yaml

```diff
@@ -0,0 +1,6 @@
+maintainer:
+  email: "hb-fe-tech@verizonmedia.com"
+capabilities:
+  site:
+    mediaTypes:
+      - banner
```

Comment:
The yaml file is only listing `banner` in the mediatypes section but `adapters/onemobile/onemobile.go` seems to can also handle video, isn't it?
```
152 func getMediaTypeForImp(impId string, imps []openrtb.Imp) openrtb_ext.BidType {
153     mediaType := openrtb_ext.BidTypeBanner
154     for _, imp := range imps {
155         if imp.ID == impId {
156             if imp.Video != nil {
157                 mediaType = openrtb_ext.BidTypeVideo <==
158             }
159             return mediaType
160         }
161     }
162     return mediaType
163 }
```

Reply:
We don't have video support, at least not for now, this check was added as defense check so the media type is reported correctly based on the response instead of statically set it  to banner type. If you think that this might break the capabilities we reported we'll remove the method.

Reply:
Prebid server supports `Banner`, `Video`, `Audio`, and `Native` as media types. If your system only supports `Banner` maybe a fair thing to do is to add a check to filter out all other media types just in case.